dom.bs

```diff
@@ -894,8 +894,8 @@ for historical reasons.
 <pre class=idl>
 [Constructor, Exposed=(Window,Worker)]
 interface EventTarget {
-  void addEventListener(DOMString type, EventListener? callback, optional (AddEventListenerOptions or boolean) options);
-  void removeEventListener(DOMString type, EventListener? callback, optional (EventListenerOptions or boolean) options);
+  void addEventListener(DOMString type, EventListener? callback, optional (AddEventListenerOptions or boolean) options = false);
```

Comment:
Yeah, exactly.  Our [implementation](https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/events/EventTarget.cpp?type=cs&q=EventTarget::addEventListener&sq=package:chromium&l=318) also works that way too - getting the dictionary defaults automatically from the bindings generator, not explicitly in code (ignoring, for now, our special treatment of passive).